styles/styles.css

```diff
@@ -0,0 +1,264 @@
+body {
+    font-family: 'Berkshire Swash', cursive;
+}
+p, nav {
+    font-family: 'Anaheim', sans-serif;
+}
+/*Header*/
+.header {
+    background-image: url(/images/promo-background.jpeg);
+    height: 420px;
+    width: 99.5%vw; 
+    color: white;
+}
+.top-container {
+    display: flex;
+}
+/*Logo*/
+.logo {
+    margin-top: 10px;
+    margin-left: 40px;
+    margin-right: -60px;
+}
+/*End of logo*/
+ul {
+    list-style-type: none;
+    display: inline-flex;
+    padding: 20px;
+    color: white;
+    font-family: 'Berkshire Swash', cursive;;
+    font-size: small;
+    margin-left: 240px;
```

Comment:
If I see an unusually large value like this, it means you're using margin to position the element, which could mean something is wrong. Additionally, I would not place such properties on an element tag (in this case `<ul>`) because it means all `<ul>` elements are affected by this. There maybe cases in the future where you don't want a `<ul>` to look like this. 1) place such properties in it's own class and 2) Perhaps find another way to position this element? (I would need to view the site to see if it makes sense or not)